src/modules/outputs/builtins/lint-commit-msg/entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+# shellcheck shell=bash
+
+function main {
+  local commit_diff
+  local commit_hashes
+  local current_branch
+  local main_branch=__envBranch__
+  local ci="${!__envCI__:-}"
+
+  current_branch="$(git rev-parse --abbrev-ref HEAD)" \
```

Comment:
@kamadorueda Decided to get this value using git instead of relying on context-dependent vars like `CI_COMMIT_REF_NAME`